src/components/events/EventForm.jsx

```diff
@@ -76,7 +84,17 @@ class EventForm extends React.Component {
         <div style={{ margin: '25px' }}>
           <DialogInput fullWidth name="eventTitle" label="Event Title" onChange={this.handleChange} value={eventTitle} />
           <div style={{ display: 'flex' }}>
-            <DialogInput style={{ marginRight: '5px' }} fullWidth name="eventDate" label="Event Date" onChange={this.handleChange} placeholder="DD/MM/YYYY" value={eventDate} />
+            <DialogInput 
+              fullWidth 
+              InputLabelProps={{ shrink: true }} 
```

Comment:
what is shrink?

Reply:
`shrink` is being used to fix the problem where the label for date text field would overlap with the placeholder when the field was not focused.

Some more info: https://material-ui.com/demos/text-fields/#limitations